src/main/java/org/commonjava/indy/service/repository/data/cassandra/CassandraStoreDataManager.java

```diff
@@ -406,7 +408,8 @@ private Map<String, String> toExtra( ArtifactStore store )
         if ( store instanceof Group )
         {
             Group group = (Group) store;
-            putValueIntoExtra( CassandraStoreUtil.CONSTITUENTS, group.getConstituents(), extras );
+            // Use admin resource addConstituentToGroup API to add the member of group instead
+            // putValueIntoExtra( CassandraStoreUtil.CONSTITUENTS, group.getConstituents(), extras );
```

Comment:
@sswguo Do you think we need to annotate this line? I removed this since it may cause the CONSTITUENTS override race condition, and introduce dirty data, we have new API could cover the members data append instead. But after removing the group update call from Indy Koji: https://github.com/Commonjava/indy/commit/8ad742a814f80586937ec527a51ed792e9e5ff48, I think this part may not be involved either anymore, there's no need to make any further change on this, WDYT?

Reply:
yeah, this should be kept as before. We just introduce that new endpoint for adding remote repo, and which does not impact the other parts, like the one creating/updating the group,  they still need this to update the members.